.github/dependabot.yml

```diff
@@ -58,4 +58,4 @@ updates:
     open-pull-requests-limit: 5
     labels:
       - "dependencies"
-      - "npm"
+      - "npm"
```

Comment:
Remove this change, if you add a new line at the end, this change should be gone.